harness/determined/pytorch/dsat/_dsat_search_method.py

```diff
@@ -749,10 +750,10 @@ def _searcher_state_tests(
         )
         logging.info(f"total slots (SearcherState, {text}): {total_slots}")
 
-        assert (
-            num_running_trials <= self.trial_tracker.max_concurrent_trials
-        ), f"running trials {num_running_trials}, limit {self.trial_tracker.max_concurrent_trials}"
-        f", {running_trials}"
+        if num_running_trials > self.trial_tracker.max_concurrent_trials:
+            logging.warn(
+                f"running trs {num_running_trials}, lim {self.trial_tracker.max_concurrent_trials}"
```

Comment:
I would prefer if the message output the name of the variables, "trs" and "lim" are not very clear out of context

Reply:
This was a temporary hack to get under the flake8 line length limits we seem to have started imposing in our pre-commit recently (only noticed since our most recent merge with `main`).  The containing `_searcher_state_tests` function is also just for testing purposes and will be moved soon.